src/autoqasm/reserved_keywords.py

```diff
@@ -0,0 +1,73 @@
+# Copied from: https://github.com/openqasm/openqasm/blob/main/source/grammar/qasm3Lexer.g4
```

Comment:
yes, although I think the link is https://github.com/openqasm/openpulse-python/blob/main/source/grammar/openpulseLexer.g4

yes, it would at least be worth separating them with comments in the code, even if we don't actually separate them into separate objects.

Reply:
I have added `waveform`, `port`, `frame` keywords already. But I'll separate them with comments. Should I separate the common ones as well like `# Common keywords between lexers` for `cal` and `defcal`